drizzle/client/0004_fresh_pestilence.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE `backupProjectInfo` (
```

Comment:
I think we should delete these two files from the journal and run another db:generate:client so that we can reduce the number of migrations happening